services/collaboration/pkg/config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"context"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+	"github.com/owncloud/ocis/v2/ocis-pkg/tracing"
+)
+
+// Config combines all available configuration parts.
+type Config struct {
+	Commons *shared.Commons `yaml:"-"` // don't use this directly as configuration for a service
+
+	Service Service `yaml:"-"`
+	App     App     `yaml:"app"`
+
+	Secret string `yaml:"secret" env:"COLLABORATION_SECRET" desc:"Used as JWT token and to encrypt access token."`
```

Comment:
We do have in other services the naming `STORAGE_PUBLICLINK_JWT_SECRET`, see that there is jwt in the envvar name, which would lead me to `COLLABORATION_JWT_SECRET`. Additionally, like in the storage_publiclink service, we have as global envvar `OCIS_JWT_SECRET` as alternative. Does this apply here too?

Reply:
Yes, I'll copy it for consistency with other services.